frontend/src/Components/Canada/AfterArrival.js

```diff
@@ -1,18 +1,59 @@
 import React from 'react';
-import NavBar from "../Misc/NavBar";
-import Footer from "../HomePage/Footer";
+import NavBar from '../Misc/NavBar';
+import Footer from '../HomePage/Footer';
+import '../../styles/AfterArrival.css';
 
-class AfterArrival extends React.Component {
+class UponArrival extends React.Component {
   render() {
     return (
       <div>
         <NavBar/>
-        <div>Nothing for after arrival yet</div>
+        
+        <div className = 'container'>
+          
+          {/* Title */}
+          <div className = 'h1-after'>After Arrival</div>
+          <hr className = 'hr-after'></hr>
+
+          {/* Step 1 */}
+          <div className = 'card border-primary'><div style = {{ width:'78vw' }}>
```

Comment:
Right now, the steps cards have a hover animation on the desktop version. Personally, I don't think it's necessary.